fideslog/api/database/data_access.py

```diff
@@ -55,6 +57,29 @@ def create_event(database: Session, event: AnalyticsEvent) -> None:
     log.debug("Event created: %s", logged_event)
 
 
+def create_user_registration_event(
+    database: Session, event: UserRegistrationEvent
+) -> None:
+    """Create a new yser registration event."""
+
+    logged_event = event.dict(exclude=EXCLUDED_ATTRIBUTES)
+    log.debug("Creating user registration from: %s", logged_event)
```

Comment:
Even while accounting for the `EXCLUDED_ATTRIBUTES`, this would write user email addresses to log output, and it would do so in conjunction with their organization.